configs/crowbar.conf

```diff
@@ -9,8 +9,7 @@
     ErrorDocument 422 /422.html
     ErrorDocument 500 /500.html
 
-    ProxyPass / http://127.0.0.1:3000/ timeout=600 Keepalive=On
+    ProxyPass / http://127.0.0.1:3000/ timeout=1500 Keepalive=On
```

Comment:
This file is only used during iniital crowbar setup from the rpm package. It is replaced with a chef-controlled file later. We can probably do without proxy_timeout here.